apps/payments/next/app/[offeringId]/success/page.tsx

```diff
@@ -48,60 +56,131 @@ export default async function Page({ params }: { params: CheckoutParams }) {
   const [contentful, cart] = await Promise.all([contentfulData, cartData]);
   /* eslint-disable @typescript-eslint/no-unused-vars */
   // const cartService = await app.getCartService();
-  const date = new Date(cart.createdAt).toDateString();
+  const date = cart.createdAt / 1000 - 86400 * 31;
```

Comment:
Note: I'm having trouble getting the localized date (see screenshots).

Reply:
I don't think any change is required to format the date. The helper functions `getLocalizedDate` and `getLocalizedDateString` already do all the formatting, we just need to make sure that `date` is in unixSeconds, since that's what's currently expected. (Which is how we get it from Stripe typically)

So in this case changing the code to `date = cart.createdAt / 1000` would suffice.

Reply:
Oh, just out of interests sake, cause I had to look this up, [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#timezone) used by `getLocalizedDateString` defaults to the runtimes default time zone.

That's something to be mindful of, since depending on whether this code is run on the server or the client, it would use the servers runtime or the clients runtime, respectively, and the two might not always be the same.